fastrag/ui/prompt_llm.py

```diff
@@ -24,6 +24,15 @@
 from fastrag.ui.utils import API_ENDPOINT, display_runtime_plot, haystack_is_ready, query
 
 DEFAULT_PROMPT = """Answer the question using the provided context. Your answer should be in your own words and be no longer than 50 words. \n\n Context: {join(documents)} \n\n Question: {query} \n\n Answer:"""
+pipeline_path = os.getenv("PIPELINE_PATH", None)
```

Comment:
what is PIPELINE_PATH?

Reply:
I had it previously to change the default prompt used in the UI, but its not crucial. I removed it